src/main/java/org/opensearch/geospatial/ip2geo/jobscheduler/Datasource.java

```diff
@@ -378,6 +408,18 @@ public boolean isCompatible(final List<String> fields) {
         return true;
     }
 
+    /**
+     * Convert long to instant
+     *
+     * This method is static so that it can be used in child class
+     *
+     * @param epochMilli the epoch milliseconds
+     * @return the instant
+     */
```

Comment:
nit: i don't think you need comments since it is very obvious from implementation/method name and it is private method.

Reply:
Can you use private method in child class?

Reply:
You can use private method in inner class

Reply:
i think you mean inner class?